torch/testing/__init__.py

```diff
@@ -27,6 +27,11 @@ def assert_allclose(actual, expected, rtol=None, atol=None, equal_nan=True):
             raise ValueError("rtol and atol must both be specified or both be unspecified")
         rtol, atol = _get_default_tolerance(actual, expected)
 
+    if actual.is_complex() and expected.is_complex():
```

Comment:
This is great for when your test is passing, but when it does not the checking part below won't work (at the very least it requires `close` variable to be set).